tests/components/flickelectric/test_flick_config_flow.py

```diff
@@ -0,0 +1,80 @@
+"""Test the Flick Electric config flow."""
+from asyncio import TimeoutError
+
+from asynctest import patch
+from pyflick.authentication import AuthException
+
+from homeassistant import config_entries, setup
+from homeassistant.components.flickelectric.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    print(result)
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.flickelectric.config_flow.FlickConfigFlow._validate_input",
```

Comment:
Don't patch our code, but the library code, ie `"homeassistant.components.flickelectric.config_flow.SimpleFlickAuth.async_get_access_token"`.

Reply:
Done :)